Pipfile

```diff
@@ -7,6 +7,8 @@ name = "pypi"
 aiobotocore = "==2.13.2"
 aiohttp = "==3.10.5"
 fqdn = "==1.5.1"
+sfapi-client = "==0.3.0"
+authlib = "==1.3.2"
```

Comment:
authlib is needed to make tokens for manual testing and will be needed in the service in the future